### DISCUSSION TITLE
DIA-NN search: setting Ox(M) and Ac(N-term) in the default panel, or in the additional options

### DISCUSSION THREAD

**Discussion by yangyangbian** [vdemichev/DiaNN#758] [Q&A]
Dear all,
I have tried to search my DIA raw files with 2 ways.
I used the same raw files and fasta file, library-free search mode.
**1) I selected Ox(M) and Ac(N-term) as variable modifications in the default panel.**
I only wrote code like this in the "additional options":
--var-mods 4
In the output file, **I identified about 773 Ox(M) peptides, 2165 Ac(N-term) peptides.**

**2) I dis-select Ox(M) and Ac(N-term) as variable modifications in the default panel.**
But, I wrote code like this in the "additional options":
--var-mod UniMod:1001,15.9949,M
--var-mod UniMod:1002,42.0106,*n
--var-mods 4
--strip-unknown-mods
In the output file, **I identified about 196 Ox(M) peptides, 1029 Ac(N-term) peptides**.

I have tried this many times, the results are quit re-producible.
**However, I cannot understand why the results are so different.**

Why do I run some tests like this?
I have run some dimethyl labeled peptides with DIA method.
Some K in the protein might be labeled with CH2O.
In all the samples, the peptides might have either 14 Da or 28 Da mass change on K (The chance is about 50-60% of peptides labeled with dimethyl tag).
Therefore, I need to set variable modification of 14 and 28 Da in my parameter.
I want to test which way to get the best identifications of dimethyl labeled peptides.

Therefore, I just simply search my normal DIA raw files with 2 options by testing the number of Ox(M) and Ac(N-term) peptides.
In theory, the results should be quite similar.
The only difference is I set variable modifications in different place.
However, the real results are really different.

Did I run something wrong?
Thanks a lot for reading!

**Comment by yangyangbian** [comment]
[report.log - variable modifications in additional options.txt](https://github.com/vdemichev/DiaNN/files/12092077/report.log.-.variable.modifications.in.additional.options.txt)
[report.log - variable modifications in default panel.txt](https://github.com/vdemichev/DiaNN/files/12092078/report.log.-.variable.modifications.in.default.panel.txt)

I attached two log files for these two searches.

It will be so great if the team or others can help me.
Thanks a lot!

**Reply by vdemichev** [reply]
The settings are not identical, the main difference is PTM scoring for n-term Ac when using the GUI options.

**Reply by yangyangbian** [reply]
Hi Vadimi, thanks a lot for your reply.
If I want to get the similar research, what shall I do in the additional panel.?

**Reply by vdemichev** [reply]
Copy settings printed at the top of the log window.
